src/PhpBrew/Build.php

```diff
@@ -103,20 +103,13 @@ public function addPkgConfigPath($path)
     public function putPkgConfigPathsEnv()
     {
         $paths = $this->getPkgConfigPaths();
-        $pathCount = count($paths);
+        $currentPkgConfigPath = getenv('PKG_CONFIG_PATH');
 
-        if ($pathCount == 0) {
-            return false;
-        } else {
-            $currentPkgConfigPath = getenv('PKG_CONFIG_PATH');
-            if ($currentPkgConfigPath !== '') {
-                $currentPaths = explode(PATH_SEPARATOR, $currentPkgConfigPath);
-                $paths = array_unique(array_merge($paths, $currentPaths));
-            }
-            putenv('PKG_CONFIG_PATH='.implode(PATH_SEPARATOR, $paths));
-
-            return true;
+        if (!empty($currentPkgConfigPath)) {
```

Comment:
It's better to keep it `currentPkgConfigPath !== false`. `0` is a valid path and cannot be ignored